src/traces/scattergeo/calc.js

```diff
@@ -40,5 +42,14 @@ module.exports = function calc(gd, trace) {
     calcMarkerColorscale(trace);
     calcSelection(calcTrace, trace);
 
+    if(len) {
+        calcTrace[0].t = {
+            labels: {
+                lat: _(gd, 'lat') + ': ',
+                lon: _(gd, 'lon') + ': '
```

Comment:
This is an interesting case here. One could argue that labels `lat` and `lon` could be configured using a layout attribute e.g.

```
  geo: {
    lonaxis: {
       hoverprefix: 'longitude' 
    },
    lataxis: {
       hoverprefix: 'latitude'
     }
  }
}
```



So I ask: where do we draw the line between configurable hover axis labels and dictionaries?



cc https://github.com/plotly/plotly.js/issues/265

Reply:
Seems to me if we were to allow configuring labels like this, it would work like the `dfltTitle` items - the translation result would be used as the default in `supplyDefaults`, but any other value provided with the figure gets used verbatim. We can't expect to translate user-provided strings, but for the most part it doesn't matter, usually translation is not about viewing other people's plots in your own language, but about making your own plot work and function the way you want it to.

Reply:
> Seems to me if we were to allow configuring labels like this, it would work like the dfltTitle items 

Good point. There's no downside to allowing `lat:`/`lon:` to be translated via dictionaries even if we add `{lat,lon}axis.hoverprefix` at some point.